cmd/osbuild-composer/config.go

```diff
@@ -116,6 +117,7 @@ func GetDefaultConfig() *ComposerConfigFile {
 		},
 		LogLevel:  "info",
 		LogFormat: "text",
+		DNFJson:   "/usr/libexec/osbuild-composer/dnf-json",
```

Comment:
I suppose it isn't.  I just really really want to make sure the default is clear :laughing:

Reply:
I removed the other ones but decided to keep this one because I think it's good for finding the default configuration values for everything in a central place.  :thinking:

Reply:
Fair. I know that we don't have any central default configuration for the worker, but if we did, it would make sense to keep the path in it as well. I think that since we don't have this, it makes sense to keep things as they are.

Thanks!